src/plugins/drawing-tool/objects/ellipse.tsx

```diff
@@ -51,7 +52,14 @@ export const EllipseComponent = observer(function EllipseComponent({model, handl
     strokeWidth={strokeWidth}
     strokeDasharray={computeStrokeDashArray(strokeDashArray, strokeWidth)}
     onMouseEnter={(e) => handleHover ? handleHover(e, model, true) : null}
-    onMouseLeave={(e) => handleHover ? handleHover(e, model, false) : null} />;
+    onMouseLeave={(e) => handleHover ? handleHover(e, model, false) : null}
+    onMouseDown={(e)=> {
+      if (handleDrag !== undefined){
+        handleDrag(e, model);
+      }
+    }}
```

Comment:
```suggestion
    onMouseDown={(e)=> handleDrag?.(e, model)}
```
and in the other places it occurs -- simpler is better in this case

Reply:
changed in all onMouseDown functions